recipes/arrow-adbc/meta.yaml

```diff
@@ -0,0 +1,203 @@
+        - libpq
+      run:
+        - libpq
+    test:
+      commands:
+        - test ! -f $PREFIX/include/adbc.h                                       # [unix]
+        - test -d $PREFIX/lib/cmake/AdbcDriverPostgreSQL/                        # [unix]
+        - test -f $PREFIX/lib/pkgconfig/adbc-driver-postgresql.pc                # [unix]
+        - test ! -f $PREFIX/lib/libadbc_driver_postgresql.a                      # [unix]
+        - test -f $PREFIX/lib/libadbc_driver_postgresql.so                       # [linux]
+        - test -f $PREFIX/lib/libadbc_driver_postgresql.dylib                    # [osx]
+
+        - if exist %LIBRARY_INC%\adbc.h exit 1                                   # [win]
+        - if not exist %LIBRARY_BIN%\adbc_driver_postgresql.dll exit 1           # [win]
+        - if not exist %LIBRARY_LIB%\adbc_driver_postgresql.lib exit 1           # [win]
+        - if not exist %LIBRARY_LIB%\cmake\AdbcDriverPostgreSQL exit 1           # [win]
+        - if not exist %LIBRARY_LIB%\pkgconfig\adbc-driver-postgresql.pc exit 1  # [win]
+
+  - name: adbc-driver-sqlite-cpp
+    script: build-cpp.sh                                                         # [not win]
+    script: build-cpp.bat                                                        # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+        - pkg-config
+      host:
+        - libsqlite
+      run:
+        - libsqlite
+    test:
+      commands:
+        - test ! -f $PREFIX/include/adbc.h                                       # [unix]
+        - test -d $PREFIX/lib/cmake/AdbcDriverSQLite/                            # [unix]
+        - test -f $PREFIX/lib/pkgconfig/adbc-driver-sqlite.pc                    # [unix]
+        - test ! -f $PREFIX/lib/libadbc_driver_sqlite.a                          # [unix]
+        - test -f $PREFIX/lib/libadbc_driver_sqlite.so                           # [linux]
+        - test -f $PREFIX/lib/libadbc_driver_sqlite.dylib                        # [osx]
+
+        - if exist %LIBRARY_INC%\adbc.h exit 1                                   # [win]
+        - if not exist %LIBRARY_BIN%\adbc_driver_sqlite.dll exit 1               # [win]
+        - if not exist %LIBRARY_LIB%\adbc_driver_sqlite.lib exit 1               # [win]
+        - if not exist %LIBRARY_LIB%\cmake\AdbcDriverSQLite exit 1               # [win]
+        - if not exist %LIBRARY_LIB%\pkgconfig\adbc-driver-sqlite.pc exit 1      # [win]
+
+  - name: adbc-driver-manager
+    script: build-python.sh                                                      # [not win]
+    script: build-python.bat                                                     # [win]
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cross-python_{{ target_platform }}                                     # [build_platform != target_platform]
+        - cython
+        - pip
+        - poetry
```

Comment:
poetry is not used anymore nowadays and can be removed?

Reply:
Thanks for catching that. Removed.

Reply:
Poetry is still listed here...